### DISCUSSION TITLE
xmake.lua lsp的警告

### DISCUSSION THREAD

**Discussion by ylingu** [Mythos-404/xmake.nvim#10] [Q&A] [answered]
使用LazyVim默认配置，打开xmake.lua时爆出一堆警告，请问作者有什么解决的办法吗？

**Comment by Mythos-404** [comment]
可以发一下你的插件配置和报错内容吗?

**Reply by ylingu** [reply]
这是报错的xmake.lua截图
![image](https://github.com/Mythos-404/xmake.nvim/assets/111283438/fef4c8e7-6316-47c7-b65b-49ff1992e145)
这是lazy-lock.json的截图
<img width="570" alt="image" src="https://github.com/Mythos-404/xmake.nvim/assets/111283438/4e41358a-b4fc-4522-b168-e4cb8b3b815f">
基本就是默认的LazyVim配置

**Reply by Mythos-404** [reply, marked as answer]
`xmake.lua`的内容是lsp的警告和插件没关系，你把插件去了也有的，你可以看看[`LLS-Addons`](https://github.com/LuaLS/LLS-Addons)的文档
